src/uiTest/resources/project.rhino.js

```diff
@@ -289,9 +302,13 @@ function Project(underlyingProject) {
     }
   };
 
-  const clickMouseInTheMiddle = function () {
-    const ideFrame = getComponentByClass('com.intellij.openapi.wm.impl.IdeFrameImpl');
-    robot.click(ideFrame);
+  this.clickMouseInTheMiddle = function () {
+    System.out.println("clickMouseInTheMiddle(): about to find ideFrame");
+    const ideFrame = getComponentByClassNoWait('com.intellij.openapi.wm.impl.IdeFrameImpl');
```

Comment:
TBH I'd consider switching to the `...NoWait` versions everywhere, i.e. pulling the await/retry logic to the higher level everywhere